configs/train_configs/dpo/valpy_dpo.yaml

```
@@ -5,7 +5,7 @@ gradient_checkpointing: true
 tokenizer_name: /model
 use_slow_tokenizer: true
 dataset_name: valpy/uf_anteater_wildchat
-max_seq_length: 4096
+max_seq_length: 2000
```

plz 2048. make the power of 2 gods happy
```suggestion
max_seq_length: 2048
```